language/diem-framework/transactional-tests/tests/prologue/fail_if_cant_pay_gas.exp

```diff
@@ -0,0 +1,4 @@
+processed 3 tasks
+
+task 2 'run'. lines 15-38:
+Error: Transaction discarded. VMStatus: status INSUFFICIENT_BALANCE_FOR_TRANSACTION_FEE of type Validation
```

Comment:
Can we print out more information in this case? Similar to how I do for `format_vm_error` in `vm_test_harness` ?

Reply:
I think something a bit more structy would be easier to grok quickly

Reply:
Hmm... this turned out to be quite tricky. `DiemVM` converts all `VMError` into `VMStatus` upon returning and a lot of information is lost in that process, so it's not possible for us to print them out here.

> I think something a bit more structy would be easier to grok quickly
I'll look into this though.

Reply:
> Hmm... this turned out to be quite tricky. DiemVM converts all VMError into VMStatus upon returning and a lot of information is lost in that process, so it's not possible for us to print them out here.

Actually, let me double check this. I noticed that functional tests were able to print out more detailed information than this so there's probably a way to do it...

Reply:
Okay, so I wasn't able to get an `VMError` back from a `VMStatus`, but it seems like printing the `VMStatus` in debug mode works well enough.